doc/decisions/record-storage.md

```diff
@@ -0,0 +1,51 @@
+# Record Storage
+
+## Problem
+
+We need a way to store session recordings.
+Sometimes it is required to run the `kdb` application with `sudo`, especially when changing system-wide configuration.
+Session recordings should be on a per-user basis. Storing them in the `user:/` namespace is not compatible with `sudo`.
```

Comment:
The first sentence is a requirement, the second sentence contradicts the first (see man page of sudo: "execute a command as another user").